README.md

```diff
@@ -109,3 +109,22 @@ jobs:
         # Provide a secret called `DOT_ENV` to append environment variables to the .env file
         dot_env: ${{ secrets.DOT_ENV }}
 ```
+
+# Personas
```

Comment:
Would add a note saying this is only valid for the latest model engine. 
Would be good if someone could import this personas and play changing that.